cm/README.md

```diff
@@ -224,6 +224,29 @@ cm-new_cm:
   env_file:
     .env
 ```
+# Post geofile
+
+In case a CM would like to post a goofile to API, this is possible.
+To do so, we strongly encourage you to use the function in the [base/BaseCM/cm_output.py](./base/BaseCM/cm_output.py) folder.
+
+```python
+# get the api url
+API_URL = os.environ.get("API_URL")
+
+def output_raster(raster_name, raster_fd):
+    """Add a raster to the api"""
+    files = {"file": (raster_name, raster_fd, "image/tiff")}
+    resp = requests.post(API_URL + "api/geofile/", files=files)
+    return resp.ok
+```
+
+def output_raster(raster_name, raster_fd):
+    """Add a raster to the api"""
+    files = {"file": (raster_name, raster_fd, "image/tiff")}
+    resp = requests.post(API_URL + "api/geofile/", files=files)
+    return resp.ok
+
+At the moment, only Geotiff is supported.
```

Comment:
@gperonato  I think you might be interested by this

Reply:
Yes, this is what I was looking for!

Reply:
Do you know what kind of object `raster_fd` is?

Reply:
this is the file you want to post opened in 'rb' mode 

here is an example from [requests documentation](https://requests.readthedocs.io/en/master/user/quickstart/)
```
url = 'https://httpbin.org/post'
files = {'file': ('report.xls', open('report.xls', 'rb'), 'application/vnd.ms-excel', {'Expires': '0'})}

r = requests.post(url, files=files)
```

Reply:
Merci beaucop pour l'aide !